src/unicode.js

```diff
@@ -376,13 +377,13 @@ function ascii(text = '') {
  * // => 'Awie imlu gurisK Awie imlu qU myry gurU ky ipAwry ]'
  */
 
-function unicode(text = '', reverse = false) {
+function unicode(text = '', reverse = false, simplify = false) {
```

Comment:
This can get annoying with ordering. I'd recommend converting the params to be an object so the definitions is 
```
function unicode({ text = '', reverse = false, simplify = false }) {

}
```
Then the spec can be `unicode(options)` and where option has those properties.

Reply:
Ideally all the public functions should be converted to this approach (not necessarily in this PR) and be released as a breaking change.

Reply:
If we want to keep it backwards compatible, a new function would be needed.